### DISCUSSION TITLE
light-mode high-contrast is not that high-contrast in practice

### DISCUSSION THREAD

**Discussion by NiloCK** [community/community#10098] [Accessibility]
Hiya. Thanks for your recent efforts in applying dark / dimmed-dark, etc themes to github.

I'm not exactly an accessibility case - my eyesight is strong - but I have a monitor with contrast issues which make regular light mode hard to use. I've had success with the dark modes on this monitor, but would prefer a light theme.

Today I discovered the high-contrast light theme. Great!

But trying it out I found that, while UI and navigation etc were quite high contrast, the **colour scheme for displaying code** was the same as in regular light mode, and **not easy reading**.

I'm sure the default theme is within regular accessibility guidelines, but it seems likely to me that users opting for high-contrast light mode probably want something higher contrast there as well.

Thanks!

**Comment by cehfisher** [comment]
Hello @NiloCK - thank you for providing this feedback! We wanted to follow up on this issue. Have the recent theme changes resolved this issue for you? If yes, could you please close out this issue? If not, what additional changes are needed? Please include screenshots and/or URLs, if applicable. Thank you!
